en/starter/static-files.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: page
 title: Serving static files in Express
+description: Understand how to serve static files like images, CSS, and JavaScript in Express.js applications using the built-in express.static middleware.
```

Comment:
Understand how to serve static files like images, CSS, and JavaScript in Express.js applications using the built-in 'static' middleware.